collection-scripts/gather-odf-provider

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Expect base collection path as an exported variable
+# If it is not defined, use PWD instead
+BASE_COLLECTION_PATH=${BASE_COLLECTION_PATH:-"$(pwd)"}
+
+INSTALL_NAMESPACES=$(oc get storagecluster -A --no-headers | awk '{print $1}')
+
+# collection path for OC commands
+mkdir -p "${BASE_COLLECTION_PATH}/odf-provider-cluster/oc_output/"
+
+# Command List
```

Comment:
you should also be adding `storageprofile` to the lists and at the same time if they don't exist the commands should fail safely (if not the same already)